README.md

```diff
@@ -2,27 +2,38 @@
 
 This is a [kubernetes addon](https://github.com/mesosphere/kubeaddons) repository which contains the Kommander addon.
 
-# Using and Releasing `kubeaddons-kommander` Repository
+# Contributing to this repository
 
-- Pre-releases are only tagged with `testing-` tags, no `stable-` tags for pre-releases.
-- After we cut a new minor GA release, we will branch that minor version off of `master` to its own `stable-` branch for maintenance (future patch releases).
-- Kommander follows a more semver style versioning and is setup to be able to support multiple major and minor versions at the same time.
+This [KEP](https://github.com/mesosphere/ksphere-platform/blob/master/keps/sig-ksphere-catalog/20200818-remove-revisions.md) changed the way we handle revision, which enables this repository to only maintain one revision per branch and removed the need to keep a flat hierarchy on master.
+
+Kommander currently is supported in three different versions, which are reporesented by their respective branches:
+
+- 1.2 being developed on `master` branch
```

Comment:
Maybe change this to 
```suggestion
- latest being developed on `master` branch
```
otherwise you have to update this file with every release, unless you're okay with that

Reply:
actually that was my plan, at least for a while :D staying abstract like "latest" and "n-1" seemed to be confusing.